web/src/components/overview/StorageSection.tsx

```diff
@@ -115,7 +115,13 @@ export default function StorageSection() {
   const availableDevices = useAvailableDevices();
   const result = useProposalResult();
 
-  if (result === undefined) return;
+  if (result === undefined) {
+    return (
+      <Content>
+        <Text>{_("Install using an advanced method.")}</Text>
```

Comment:
What about using "configuration" instead of "method"? I think that's easier to relate to what really happened.